game-core/src/main/java/tools/image/FileOpen.java

```diff
@@ -10,16 +10,6 @@
 public class FileOpen {
   private File file = null;
 
-  /**
-   * Default Constructor.
-   *
-   * @param title The title of the JFileChooser.
-   * @exception java.lang.Exception
-   *            ex
-   *            Creates a file selection dialog starting at the current
-   *            working directory. Filters out all non-txt files and
-   *            handles possible file load errors.
```

Comment:
This description is no longer accurate, so I just removed the entire Javadoc as no other constructor in this class was documented.